### PR TITLE
adjust to ovs-cni 0.4 deployment, including marker

### PR DESCRIPTION
**Pull request by phoracek**
Since 0.3 ovs-cni includes not only plugin container, but now also
marker. Marker exposes bridges as node's resources.

Since their pod now carries both, it was renamed from ovs-cni-plugin
to ovs-cni. Because of that, deprovision script now removes also
deprecated ovs-cni-plugin daemon set.

With 0.4 we also moved ovs-vsctl daemon set functionality to ovs-vsctl
daemon set init container.

```release-note
deploy ovs-cni marker in order to expose bridges as node resources
```

### CODE REVIEW

**Comment by phoracek** [comment]
@SchSeba @leongold could you please take a look?

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
@rthallisey @lukas-bednar @SchSeba hi, would you please take a look?

File: roles/network-multus/templates/openshift-ovs-cni.yml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  namespace: {{ network_namespace }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ ovs_cni_marker_service_account }}
+  namespace: {{ network_namespace }}
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```
[the remark this answers is not in the document]
Comment (phoracek) [reply roles/network-multus/templates/openshift-ovs-cni.yml]: Oh that's a great idea! But we should change it in ovs-cni repo first and just reflect here do avoid out-of-sync code. I will take a look at it.

[the remark this answers is not in the document]
Comment (phoracek) [reply roles/network-multus/templates/openshift-ovs-cni.yml]: Addressed in: https://github.com/kubevirt/ovs-cni/pull/66

**Comment by phoracek** [comment]
ci test please

Looks like it failed on unrelated storage issue.

**Comment by phoracek** [comment]
@SchSeba @booxter could you please take a look? Should be ready.

**Comment by phoracek** [comment]
Please don't merge yet, forgot to add deprovision of the removed ovs-vsctl daemon set.

**Comment by phoracek** [comment]
@booxter @SchSeba @lukas-bednar ok, now it should be ready. Could you please take a look?

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
@rthallisey @lukas-bednar @SchSeba @booxter now it should be really ready. Please take a look.

**Comment by lukas-bednar** [comment]
LGTM!

**Pull request merged**